tests/sqlparser_clickhouse.rs

```diff
@@ -553,6 +554,11 @@ fn parse_select_star_except() {
     clickhouse().verified_stmt("SELECT * EXCEPT (prev_status) FROM anomalies");
 }
 
+#[test]
+fn parse_select_parametric_function() {
+    clickhouse().verified_stmt("SELECT quantile(0.5)(x) FROM t");
+}
```

Comment:
Could we extend the test asserting that the parameter and arguments lists do indeed show up in the `parameters` and `args` fields respectively (i.e not vice versa for example). Also we can include multiple items in both lists e.g `quantile(0.5,0.6)(x,y)`